### ISSUE TITLE
Code example tabs missing ARIA roles

### ISSUE THREAD

**Issue by TrebBrennan** [govau/designsystem#58]
RE: #54

The tabs for component code examples (HTML/React) are missing a programmatic way to determine state. (A screenreader does not know what the link does or what has opened or closed elsewhere on the page)

See. `.tab-item--active`

# Notes.
- [WCAG example of tabs](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/tabs/tabs-2/tabs.html)

<img width="992" alt="screen shot 2018-04-05 at 11 10 22 am" src="https://user-images.githubusercontent.com/15899197/38342751-9314979e-38c3-11e8-867a-478989d71263.png">

**Comment by TrebBrennan** [comment]
Keyboard behaviour for tabs is also odd:

Tabbing to 'HTML' and pushing enter will cause the next focus state to be 'copy' rather than 'React', which is unexpected.

![tabs-tab-order](https://user-images.githubusercontent.com/15899197/38349132-f5d2c200-38e8-11e8-9daa-e6588e8bbab5.gif)
